### PR TITLE
Proper message when registering an already registered snap

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

**Comment by sergiusens** [comment]
Depends on #623

**Comment by elopio** [comment]
wow, all green. You are in a streak.

File: integration_tests/test_store_register.py [hunk 1]
```diff
@@ -55,3 +57,15 @@ def test_failed_registration_already_registered(self):
...
+        # The snap name is already registered.
+        error = self.assertRaises(
+            subprocess.CalledProcessError,
+            self.register, self.reserved_name)
+        expected = (
+            'The name {0!r} is reserved.\n\n'
+            'If you are the publisher most users expect for {0!r} '
+            'then please claim the name at').format(self.reserved_name)
+        self.assertThat(str(error.output), Contains(expected))
+        self.assertThat(str(error.output), Contains('register-name-dispute'))
```
Comment (elopio) [inline integration_tests/test_store_register.py]: I'm not sure how to make a reserved name in staging.
We can get @nataliabidart to do reserve for us this name, or we can use one that's currently reserved.

Comment (elopio) [inline integration_tests/test_store_register.py]: oh, cool, you are using bash, I didn't see the diff up there ^_^.

**Comment by elopio** [comment]
:+1:

**Comment by elopio** [comment]
I ran the test with fake and staging servers.

**Pull request merged**